src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
+
+    protected String by;
+
+    public Deadline(String description, String by, Boolean isDone) {
+        super(description, isDone);
+        this.by = by;
+    }
+
+    @Override
+    public String toString() {
+        return "[D] " + super.toString() + " (by: " + by + ")";
+    }
+}
```

Comment:
Github will get angry and complain if you don't leave a new line at the end of each file